NEWS.md

```diff
@@ -1,5 +1,12 @@
 # usethis (development version)
 
+* `pr_*()` functions automatically refresh RStudio's git pane (#706).
+
+* `pr_fetch()` can optionally finish any PR, not just the current (#1040).
+
+* `pr_pause()` and `pr_fetch()` now automatically pull to get latest changes 
+  (#959, #960).
```

Comment:
I will not be surprised if we learn there's a few edge cases here, but it looks fine, just to read it.